index.js

```diff
@@ -711,6 +711,15 @@ class UserAgent {
     };
 
     /**
+     * Return the desired value for the X-Thorn header.
+     *
+     * @return {string} The value of the X-Thorn header.
+     *
+     * @private
+     */
+    _xthorn = () => `Agent-${this.username}`;
```

Comment:
That level of complexity is not worth it for the tiny level of information hiding (IMO). Those names might already show up in the database already, so it's not like we can completely obscure this anyways.